LondonBritishLibrary/orient/BLorient2083.xml

```diff
@@ -0,0 +1,2474 @@
+                                                </msItem>
+                                                <colophon xml:lang="gez" xml:id="coloph3">
+                                                    <locus target="#218ra"/>
+                                                    <note>Colophon containing the names of the scribe
+                                                        <persName role="scribe" ref="PRS14424TasfaIyasus">Tasfā ʾIyasus</persName> and of
+                                                        <persName role="patron" ref="PRS14423Malakotawit">Malakotāwit</persName>, who commissioned 
+                                                        the manuscript.</note>
+                                                </colophon>
+                                            </msItem>
+                            </msItem>
+                      </msItem>
+                    </msContents>
+                    <physDesc>
+                        <objectDesc form="Codex">
+                            <supportDesc>
+                                <support>
+                                    <material key="parchment"/>
+                                </support>
+                                <extent>
+                                    <measure unit="leaf">218</measure>
+                                    <dimensions type="outer" unit="mm">
+                                        <height>350</height>
+                                        <width>307</width>
+                                    </dimensions>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="22"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Neat gʷǝlḥ script, decreasing in height from 6 mm at the beginning to about 4 mm at the end of the manuscript.</desc>
+                                <date notBefore="1694" notAfter="1706" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus from="137r" to="177r"/>
+                                    <desc type="Comment">Short notes summarizing the main events of the Passion in the upper margins of the 
+                                        readings for Good Friday.</desc>
+                                </item>
+                                <item xml:id="a2">
+                                    <locus from="218ra" to="218vb"/>
+                                    <desc type="Supplication">Benediction</desc>
```

Comment:
supplications are usually scribal notes, not additiones, here we have a Prayer or a GuestText (salota burake)